docs/integrations/README.md

```diff
@@ -180,6 +180,7 @@ For more information about the grading system, see [Product Release Stages](http
 | [Zenloop](sources/zenloop.md)                                                               | Alpha                | Yes                 |
 | [Zoho CRM](sources/zoho-crm.md)                                                             | Alpha                | No                  |
 | [Zoom](sources/zoom.md)                                                                     | Alpha                | No                  |
+| [Zoom low-code CDK](sources/zoom-lc-cdk.md)                                                        | Alpha                | No                  |
```

Comment:
you can remove this line in favor of the other one Zoom.

Reply:
Done. Thank you.